### ISSUE TITLE
xft display glitch when dragging windows

### ISSUE THREAD

**Issue by kwshi** [brndnmtthws/conky#935]
**Issue**

When `use_xft` is set to `true`, font rendering glitches when other windows are dragged around.  Specifically, when windows are dragged around, conky text _slides_ downwards and leaves a white trail.  See attached GIF for demonstration.

![conky-xft-bug](https://user-images.githubusercontent.com/3887189/73692073-7293b200-4688-11ea-9997-780d11068a4c.gif)

The issue does not arise when `use_xft` is disabled.

**Information**

OS: NixOS 19.09
Conky: 1.11.5

```
conky.config = {
  gap_x = 16,
  gap_y = 40,
  alignment = "tl",
  update_interval=1,
  use_xft = true,
  font = "RobotoMono:size=11",
};

conky.text = "..."
```

**Comment by kwshi** [comment]
Update: using `double_buffer = true` makes the white trails disappear, but the text still drifts when windows are moved around:
![conky-db-xft-glitch](https://user-images.githubusercontent.com/3887189/73692442-2b59f100-4689-11ea-81a2-8c94acfc82e1.gif)

**Comment by rubyFeedback** [comment]
I can't help you, but while this is evidently a bug, it is also quite epic - it's like conky on steroids, dancing and jiggling and having fun here! \o/

**Comment by kwshi** [comment]
This issue appears to be fixed now, at some point before version 1.12.1; I can no longer reproduce.  So, yay!
